cypress/fixtures/edgeHosts.json

```diff
@@ -0,0 +1,6 @@
+{
+  "results": [],
```

Comment:
Should we add some edge hosts here?

Reply:
i mean, we don't need it for anything right now, it could be added in case it's needed for the tests
but i can add a few of them if you think it's better that way